src/components/structures/auth/ForgotPassword.js

```diff
@@ -205,14 +198,14 @@ module.exports = React.createClass({
             );
         } else if (this.state.progress === "complete") {
             resetPasswordJsx = (
-                <div className="mx_Login_prompt">
+                <div>
                     <p>{ _t('Your password has been reset') }.</p>
                     <p>{ _t('You have been logged out of all devices and will no longer receive push notifications. ' +
                         'To re-enable notifications, sign in again on each device') }.</p>
                     <input className="mx_Login_submit" type="button" onClick={this.props.onComplete}
                         value={_t('Return to login screen')} />
                 </div>
-            );
+            )
```

Comment:
surprised the linter isn't complaining about this

Reply:
It's an excluded file... 🙃 I'll fix and send a separate PR to lint these files going forward.